Modules/Scripted/SegmentStatistics/SegmentStatistics.py

```diff
@@ -332,9 +333,9 @@ def setup(self):
         self.doneButton.connect('clicked()', lambda: self.done(1))
 
     def _addPluginOptionWidgets(self):
-        description = "Edit segment statistics plugin parameters:"
+        description = _("Edit segment statistics plugin parameters:")
         if self.pluginName:
-            description = "Edit " + self.pluginName + " plugin parameters:"
+            description = _("Edit {pluginName} plugin parameters:").format(pluginName=self.pluginName)
```

Comment:
We need to introduce a separate plugin Title property, which is displayed on the GUI and it is translated, while plugin name is not translatable.